### ISSUE TITLE
Water flow

### ISSUE THREAD

**Issue by JanChrisKlein** [jomjol/AI-on-the-edge-device#277]
Hi,

some questions concerning the water flow (rate).
The received values are rather small: 0.001658 for example. What is the unit? Is it possible to change it?
1.658 l/min would seem realistic but I can't get Domoticz to convert to that.

How is the rate calculated? Values are reported only once every 3 minutes so is it the average of used water during these 3 minutes?

**Comment by jomjol** [comment]
1) The unit is typically m³ = 1000l. So 0.001658 is ineed 1.658l.
2) The rate is (current - last) / time --> so you're right it is the average of the 3 minutes

The unit cannot be changed directly, this is just not in the code. You could do that indirectly, by using the decimal shift option. With this you can change the unit to liter by shifting 3 digit (e.g. 10.00165 --> 10001.65). This would also diretly influence the rate.

**Comment by JanChrisKlein** [comment]
Okay. So I set "DecimalShift" to 3 which multiplied "value" by 1000. "rate" went to 0. Just to be sure I switched "DecimalShift" to -3, again rate remained at 0 (which is plausible). I then changed "DecimalShift" to 3 again and "MaxRateValue" from 0.1 to 50 and other values: rate = 0.000000 ...
I then changed everything back to as it was. rate= 0.00000 ;)
Any idea? In the config-file everything seems to be correct.

Christoph
